dev-python/ipcalc/ipcalc-1.99.0.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="IP subnet calculator"
+HOMEPAGE="https://pypi.python.org/pypi/ipcalc/"
+SRC_URI="mirror://pypi/i/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="amd64 x86"
```

Comment:
Don't add new version with stable keywords.

Reply:
Version 1.99.0 is been released on Jul 14, 2015, I think that we can consider it stable.
I think to set stable also for your email about few ebuilds with stable condition and in this case I think that could  be correct set soon stable for this ebuild.
However, I haven't problem to change this.

Reply:
In Gentoo new versions always start in testing. It's not only about upstream bugs but also testing integration.

Reply:
Ok, I proceed with change. Thanks for clarification

Reply:
Last question... so when we can consider stable an ebuild ? is there a minimal time window or some other algorithm ?

Reply:
The common rule is '30 days in ~arch and no major regressions'. This can be hastened e.g. when there's a security fix included, or delayed more e.g. when the package has many dependencies or is likely to break stuff.